docs/user-guide/deploying-your-docs.md

```diff
@@ -30,27 +30,29 @@ That's it! Behind the scenes, MkDocs will build your docs and use the
 Use `mkdocs gh-deploy --help` to get a full list of options available for the
 `gh-deploy` command.
 
-Be aware that you will not be able to review the built site before it is pushed
-to GitHub. Therefore, you may want to verify any changes you make to the docs
-beforehand by using the `build` or `serve` commands and reviewing the built
-files locally.
+Please note that you will not be able to review the built site before it is
+deployed to GitHub. Instead, you may want to verify any changes you made by
+using the `build` or `serve` commands and reviewing the built files locally
+before running `gh-deploy`.
 
 WARNING:
-You should never edit files in your pages repository by hand if you're using
-the `gh-deploy` command because you will lose your work the next time you
-run the command.
+The site files in the `gh-pages` branch are **automatically generated** and
+updated when using the `gh-deploy` script.  
+As such, you should never make any changes to the site files of your GitHub
+Pages site by hand. Doing so will cause you to lose these changes the next time
+the `gh-deploy` script or command is called.
 
 WARNING:
 If there are untracked files or uncommitted work in the local repository where
-`mkdocs gh-deploy` is run, these will be included in the pages that are deployed.
+`mkdocs gh-deploy` is run, these will be included in the deployed Pages site.
 
-### Organization and User Pages
+### User and Organisation Pages
```

Comment:
“Organization“ with a z to remain consistent

Reply:
The following GitHub feature may be handy for this! When used, your suggested changes can be approved/accepted right from the GitHub PR page, and will also appropriately and accordingly show "HumbleDeer accepted these proposed changes" (or something in that sense). "Outdated" will no longer be shown in such cases.

![image](https://github.com/mkdocs/mkdocs/assets/16231288/121a6f39-a2ba-4d8c-8843-020f8d4d6db5)